lib/components/NewMultisigWallet.js

```diff
@@ -49,6 +87,16 @@ class NewMultisigWallet extends PureComponent {
       >
         <BoxGrid rowClass="p-sm-2" colcount={1}>
           <Header type="h2">Multisig</Header>
+          <Text>Please plug in hardware wallet</Text>
```

Comment:
I'm probably missing this somewhere (and I haven't set up a ledger to test) but is there a conditional to check if the hardware wallet is plugged in or not or is this always displayed?